falco.yaml

```diff
@@ -0,0 +1,121 @@
+package:
+  name: falco
+  version: 0.36.1
+  epoch: 0
+  description: Cloud Native Runtime Security
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - clang-16
+      - gcc
+      - cmake
+      - make
+      - git
+      - bash
+      - perl
+      - linux-headers
+      - autoconf
+      - automake
+      - m4
+      - libtool
+      - elfutils-dev
+      - libelf
+      - libelf-static
+      - patch
+      - binutils
+      - libbpf
+      - openssl
+      - openssl-dev
+      - yaml-dev
+      - c-ares
+      - c-ares-dev
+      - protobuf
+      - protobuf-dev
+      - protobuf-c-dev
+      - re2
+      - re2-dev
+      - zlib-dev
+      - libcurl-openssl4
+      - llvm16
+      - abseil-cpp
+      - abseil-cpp-dev
+      - abseillib
+      - jq-dev
+      - curl-dev
+      - grpc
+      - grpc-dev
+      - icu
+      - icu-dev
+      - yaml-cpp
+      - yaml-cpp-dev
+      - systemd-dev
+      - libsystemd
+      - libzstd1
+      - zstd
+      - zstd-dev
+      - libtbb-dev
+
+vars:
+  RULES_VERSION: "2.0.0"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/falcosecurity/falco
+      tag: ${{package.version}}
+      expected-commit: 9eb611609a2876a5f5a5378e0613f0ff767f0d42
+      recurse-submodules: true
+
+  - runs: |
+      # Replace the find_dependency with find_package macro for newer cmake, otherwise it will fail.
+      sed -i 's/find_dependency(Protobuf CONFIG)/find_package(Protobuf CONFIG)/' /usr/lib64/cmake/grpc/gRPCConfig.cmake
+
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/etc/falco
+      install -Dm755 ./falco.yaml "${{targets.destdir}}"/etc/falco/falco.yaml
+      sed -e 's/time_format_iso_8601: false/time_format_iso_8601: true/' < "${{targets.destdir}}"/etc/falco/falco.yaml
+
+  - working-directory: /home/build/rules
+    pipeline:
+      - uses: git-checkout
+        with:
+          repository: https://github.com/falcosecurity/rules
+          tag: falco-rules-${{vars.RULES_VERSION}}
```

Comment:
This is going to require manual updates since the wolfibot won't know how to update it.

Would it make sense to have `falco-rules.yaml` as a separate package, that only contained these rules definitions?

Reply:
Are the rules something that we expect users of the package to modify? In addition to what @imjasonh said, if we make this subpackage it's easier for folks to update it themselves.

Reply:
> Are the rules something that we expect users of the package to modify?

Yep, correct. But Falco also provides default rules.

> Would it make sense to have falco-rules.yaml as a separate package

It does make sense.

I was thinking if we should provide `falco_rules` by default with the `falco` package itself. If so, we should make separate `falco-rules` package; otherwise a sub-pkg is way to go.

Should we provide this as a sub-pkg?

Am I allowed to use `patch` in sub-pkg?

Reply:
You can use patch in a subpackage.

But since it comes from a separate repo, I think falco-rules belongs as a toplevel package, so they can be updated automatically